src/Compilers/Core/Portable/Operations/OperationCloner.cs

```diff
@@ -1,13 +1,24 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Immutable;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Operations
 {
-    internal abstract class OperationCloner : OperationVisitor<object, IOperation>
+    internal sealed class OperationCloner : OperationVisitor<object, IOperation>
     {
-        protected T Visit<T>(T node) where T : IOperation
+        private static readonly Lazy<OperationCloner> _lazyInstance = new Lazy<OperationCloner>(() => new OperationCloner());
```

Comment:
I don't think we need OperationCloner to be lazy? since it doesn't have much fields? Lazy might be more expansive memory-wise?